guides/GOVERNANCE.md

```diff
@@ -1,2 +0,0 @@
-See our governance process in Confluence
-https://confluence.cms.gov/display/HCDSG/Governance
```

Comment:
https://confluence.cms.gov/display/HCDSG/Design+system+governance

I use the share button to get the following link, and I'm hoping it means it's a permalink:
https://confluence.cms.gov/x/LYWbGQ

Reply:
Wondering if it should just go in the README or if it's important that it's in it's own GOVERNANCE file